internal/identity/gitlab.go

```diff
@@ -19,8 +19,7 @@ import (
 
 const (
 	defaultGitLabProviderURL = "https://gitlab.com"
-	revokeURL                = "https://gitlab.com/oauth/revoke"
```

Comment:
this is now being built from the open-id connect well known rest endpoint now?

Reply:
Yeah